### ISSUE TITLE
Incompatible version

### ISSUE THREAD

**Issue by Growlert** [dalibo/pg_log_authfail#7]
Hello!
My version of postgres is 10. I came through all the instruction and after editing postgresql.conf then try to start postgres, it's giving me the error: 'Incompatible library.....: incompatible versions. Server version: 10, library version: 9.2. How can I get aroung this issue?

**Comment by darold** [comment]
Hi,

It seems that you have compiled the extension using the 9.2 libraries. What does `pg_config --version` return on the same environment you've compiled pg_log_authfail?

**Comment by Growlert** [comment]
You are right, pg_config --version shows this string: PostgreSQL 9.2.24. Don't know why command 'yum install postgresql-devel' (I'm using Centos 7.4)  downloads and installs the old version of pg_config. 
Can you tell me how to install valid version of pg_config for my Postgres?

**Comment by darold** [comment]
Perhaps this is because you use the 7.4 Centos repository to install libraries and they are only give v9.2? I don't know but the better way is to exclude PostgreSQL to be updated/downloaded from the Centos repository and to use the official PGDG repository. See instruction at http://yum.postgresql.org/

Otherwise if you already have pg_config on your system, let's says in `/usr/pgsql-10/bin/pg_config` just add this directory in your path: `export PATH=/usr/pgsql-10/bin:$PATH`

**Comment by Growlert** [comment]
Indeed, folder pgsql-10 was in mentioned path, but missing some necessary subfolders\files, such as include, pgxs. Then I run this command 'yum install postgresql10-devel' and those folders appeared, unfortunately the path to postgres data folder was changed to default location, but I fixed it and then did this 'export PATH=/usr/pgsql-10/bin:$PATH'. After that, I run commands 'make' and 'make install', restart postgres with new added lines according to instruction and finally it's working now! 
Very useful module, thanks for your support!

**Comment by Growlert** [comment]
I found 1 mismatch in your instruction, string 'filter' in fail2ban jail.conf must be the same as the conf name in 'filter.d' folder without '.conf' extension
